ipfs.io-theme/static/js/stars.js

```diff
@@ -34,23 +40,32 @@ window.StarChart = (function () {
     return !(rect.bottom < 0 || rect.top - viewHeight >= 0);
   }
 
+  function removeItem(list, item) {
+    list.splice(list.indexOf(item), 1);
+  }
+
   function StarChart(div, pixi) {
     this.div = div;
     this.pixi = pixi;
+
+    // hack to disable console message (https://github.com/pixijs/pixi.js/issues/1900)
     this.pixi.utils._saidHello = true;
-    if (window.navigator.userAgent.indexOf('Macintosh') !== -1 || window.navigator.userAgent.indexOf('Windows') !== -1) {
-      this.renderer = this.pixi.autoDetectRenderer(800, 600, {antialias: true, transparent: true, resolution: _PIXEL_RATIO});
-    } else {
-      this.renderer = new this.pixi.CanvasRenderer(800, 600, {antialias: true, transparent: true, resolution: _PIXEL_RATIO});
-    }
+
+    this.renderer = new pixi.CanvasRenderer(800, 600, {
```

Comment:
Yes indeed, it actually has better cross-platform support than the WebGL renderer, which is why Pixi.JS treats it as a fallback [(docs)](http://pixijs.download/dev/docs/PIXI.html#.autoDetectRenderer), but in this case it runs and looks better because lines happen to be WebGL's weak point.